echopype/calibrate/api.py

```diff
@@ -139,6 +148,10 @@ def compute_Sv(echodata: EchoData, **kwargs) -> xr.Dataset:
     The current calibration implemented for EK80 broadband complex data
     uses band-integrated Sv with the gain computed at the center frequency
     of the transmit signal.
+
+    The returned xr.Dataset will contain the variable water_level from the
+    EchoData object provided, if it exists. If water_level is not returned,
+    it must be set using EchoData.update_platform().
```

Comment:
```suggestion
    The returned xr.Dataset will contain the variable `water_level` from the
    EchoData object provided, if it exists. If `water_level` is not returned,
    it must be set using `EchoData.update_platform()`.
```

We currently have some inconsistency in how variables in docstrings are written (with or without the back ticks  and also whether it is double or single). We'll have to fix those, but before then we can fix small ones as we go.

Reply:
@leewujung thank you for letting me know that we will be using back ticks from now on. I will be sure to stick with this in the future. In the meantime, I will commit the changes that you suggested.